docs/builddocslocally.md

```diff
@@ -0,0 +1,85 @@
+## Building the documentation locally

```

Comment:
We should start headings at the first level, can you please adapt the headings level to start at the first level, and also change the others to not miss any heading level in between?

Reply:
Yeah sure, I've changed the headings level and others as per markdownlint rules..